src/tests/ftest/util/dfuse_utils.py

```diff
@@ -84,14 +85,15 @@ def set_dfuse_cont_param(self, cont, display=True):
 class Dfuse(DfuseCommand):
     """Class defining an object of type DfuseCommand"""
 
-    def __init__(self, hosts, tmp, dfuse_env=False):
+    def __init__(self, hosts, tmp, log_file=None, dfuse_env=False):
```

Comment:
This change will cause a problem in https://github.com/daos-stack/daos/blob/master/src/tests/ftest/soak/soak.py#L408.   If the new, optional, positional argument was added to the end of the argument list, this could be avoided.  Alternatively, `soak.py` should be updated.

Reply:
soak.py is updated.  It always seems odd to me how optional args are used positionally.  I'll switch these two around to be on the safe side.

Reply:
Sorry I missed the update to `soak.py`.